Configurateur/front/lib/components/container.dart

```diff
@@ -0,0 +1,124 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:front/screens/company-profil/container-profil.dart';
+import 'package:front/services/storage_service.dart';
+import 'package:go_router/go_router.dart';
+
+class CtnList {
```

Comment:
we don't have a limit on the line size so why not use "ContainerListData" for example which is more explicit

Reply:
change